lib/gui.py

```diff
@@ -235,22 +300,43 @@ def make_wordlist_gen(main_window):
     min = tk.StringVar(value=1)
     max = tk.StringVar(value=100)
 
-    scan_label = tk.Label(wordlist_gen_frame, text="Select Wordlist Generator Options", bg=DEFAULT_BG_COLOR, fg="#FFF")
+    scan_label = tk.Label(
+        wordlist_gen_frame,
+        text="Select Wordlist Generator Options",
+        bg=DEFAULT_BG_COLOR,
+        fg="#FFF",
+    )
     scan_label.pack(fill=tk.X, expand=True)
 
-    def start_wordlist_gen(url: str, file: str, min: int, max: int):
-        # TODO: validate input
-        generator = WordlistGenerator()
-        res = generator.gen(url, file, int(min), int(max))
-        if not res:
-            messagebox.showerror("Error", "Something went wrong")
-        results = generator.results
-        if len(results) == 0:
-            result_label.insert(tk.END, "Generation finished, no keywords found")
+    def validate_wordlist_input(url: str, file: str, min: int, max: int) -> bool:
+        if url and validate_url(url) is False:
+            messagebox.showerror("Error", "Privided url is invalid.")
+        elif file and isfile(file) is False:
+            messagebox.showerror("Error", "Provided filepath is invalid.")
+        elif min.isdigit() is False or min.isdigit() is False:
+            messagebox.showerror("Error", "Min and max have to be digits.")
+        elif int(min) > int(max):
+            messagebox.showerror(
+                "Error", "Min has to be smaller than max or equal to max."
+            )
         else:
-            result_label.insert(tk.END, "\n".join(results))
+            return True
 
-    entries = [
+    def start_wordlist_gen(url: str, file: str, min: int, max: int):
+        wordlist_result.delete("1.0", "end-1c")
+
+        if validate_wordlist_input(url, file, min, max):
+            generator = WordlistGenerator()
+            res = generator.gen(url, file, int(min), int(max))
```

Comment:
`res` and `results` is confusing. Maybe `res` -> `succeeded`?
then the next line would be more understandable `if not succeeded:`